src/mon/OSDMonitor.cc

```diff
@@ -922,10 +922,12 @@ OSDMonitor::update_pending_pgs(const OSDMap::Incremental& inc)
     pending_creatings.pgs.erase(pg);
   }
   pending_created_pgs.clear();
-  // PAXOS_PGMAP is less than PAXOS_OSDMAP, so PGMonitor::update_from_paxos()
-  // should have prepared the latest pgmap if any
-  mon->pgservice->maybe_add_creating_pgs(creating_pgs.last_scan_epoch,
-					 &pending_creatings);
+  if (!osdmap.test_flag(CEPH_OSDMAP_REQUIRE_LUMINOUS)) {
```

Comment:
Ah, that sounds right. The  dueling monitor and osdmap LUMINOUS flags just keep jumbling up in my head.